vue.config.js

```diff
@@ -18,16 +18,22 @@ module.exports = {
       rules: [
         {
           test: /\.ts|js|vue$/,
-          enforce: "pre",
-          use: ["source-map-loader"],
+          enforce: 'pre',
+          use: ['source-map-loader']
+        },
+        {
+          test: /@empathyco\/x-components/,
+          exclude: /\.css$/,
+          enforce: 'pre',
+          use: ['babel-loader']
```

Comment:
This is much easier. Vue CLI already adds a babel loader for JS files. It also exposes a transpileDependencies option which you can configure with `transpileDependencies: ['@empathyco/x-components']` so it transpiles x-components.